tests/init/neg/enum-desugared.scala

```diff
@@ -1,35 +0,0 @@
-package example
```

Comment:
Should we move these tests to pos instead of deleting them entirely?

Reply:
Definitely